sync-core/src/test/java/com/cloudant/common/CouchTestBase.java

```diff
@@ -27,6 +28,23 @@
  */
 public abstract class CouchTestBase {
 
+
+    static {
+        try {
+            if (Misc.isRunningOnAndroid()) {
```

Comment:
Bit of documentation for this please - you explained it verbally but it would be useful to have it written down somewhere